.github/workflows/flowzone.yml

```diff
@@ -95,6 +95,16 @@ on:
         type: string
         required: false
         default: registry.npmjs.org
+      ignore_versionbot_branch:
```

Comment:
I think if we have included versionist functionality we should always ignore versionbot and not even give the option? It makes for fewer flows we need to test.